README.md

```diff
@@ -44,11 +44,11 @@ $ composer require charcoal/contrib-sitemap
 #### Required
 
 -   **[PHP 5.6+](https://php.net)**: _PHP 7_ is recommended.
--   **[charcoal/app][charcoal-app]**: ^4.0
--   **[charcoal/core][charcoal-core]**: ^4.0
--   **[charcoal/factory][charcoal-factory]**: ^4.0
--   **[charcoal/object][charcoal-object]**: ^4.0
--   **[charcoal/translator][charcoal-translator]**: ^4.0
+-   **[charcoal/app][charcoal/app]**: ^4.0
+-   **[charcoal/core][charcoal/core]**: ^4.0
+-   **[charcoal/factory][charcoal/factory]**: ^4.0
+-   **[charcoal/object][charcoal/object]**: ^4.0
+-   **[charcoal/translator][charcoal/translator]**: ^4.0
```

Comment:
This is the markdown link shorthand:

```suggestion
-   **[charcoal/app]**: ^4.0
-   **[charcoal/core]**: ^4.0
-   **[charcoal/factory]**: ^4.0
-   **[charcoal/object]**: ^4.0
-   **[charcoal/translator]**: ^4.0
```

Also, line 10 needs to be updated accordingly:

> ```
> A [Charcoal][charcoal-app] service for generating a sitemap.
> ```